src/mgr/ActivePyModules.h

```diff
@@ -193,6 +194,27 @@ class ActivePyModules
     return modules.at(name)->finisher;
   }
 
+  void inc_mod_finisher_cnt(const std::string &name) {
+    if (mod_finisher_cnt.find(name) != mod_finisher_cnt.end()) {
+      mod_finisher_cnt[name]++;
```

Comment:
The default construct of `int` would be `0`. You can simply keep this single statement as the implementation for this method.

Reply:
I always struggle to work out when that will be true in STL containers. I guess when looking up operator[] it says that it will default-construct the element, but how do you remember that's true in this instance? I know I've gotten into trouble with similar logic in the past... :/

Reply:
In general, if you do not value initialize a primitive type then it will have undefined value. There are exceptions like static duration variables.